base/src/test/kotlin/proguard/analysis/DominatorCalculatorTest.kt

```diff
@@ -381,6 +382,98 @@ class DominatorCalculatorTest : FreeSpec({
         )
     }
 
+    "Try block at start of code" {
+        val code = byteArrayOf(
+                Instruction.OP_BIPUSH, 42, // 0  bipush 42
+                Instruction.OP_BIPUSH, 43, // 2  bipush 43
+                Instruction.OP_GOTO, 0, 7, // 4  goto 11 (+7)
+                Instruction.OP_BIPUSH, 44, // 7  bipush 44
+                Instruction.OP_BIPUSH, 45, // 9  bipush 45
+                Instruction.OP_BIPUSH, 46, // 11 bipush 46
+                Instruction.OP_RETURN //      13 return
+        )
+        val exceptions = arrayOf(ExceptionInfo(0, 4, 7, 1))
```

Comment:
Nice scenarios! Can you provide some comments in the test indicating what the exception info numbers are in the above instructions (start/end offset and catch offset)?